data/lang-en.csv

```diff
@@ -380,4 +380,8 @@ CategoryCode.GOVT.Description,"Government-run websites, including military sites
 CategoryCode.COMM.Description,Sites of commercial services and products.,
 CategoryCode.CTRL.Description,Benign or innocuous content used as a control.,
 CategoryCode.IGO.Description,"Sites of intergovernmental organizations, such as the United Nations.",
-CategoryCode.MISC.Description,Sites that haven't been categorized yet.,
\ No newline at end of file
+CategoryCode.MISC.Description,Sites that haven't been categorized yet.,
+TestResults.Details.Websites.LikelyBlocked.Content.Paragraph,"{{WebsiteURL}} is likely blocked by means of {{BlockingReason}}.
+
+Note: False positives can occur. Learn more [here](https://ooni.io/nettest/web-connectivity/).",
+TestResults.Details.Websites.Reachable.Content.Paragraph,"{{WebsiteURL}} is accessible.",

```

Comment:
How does a CSV deal with newlines? I am surprised the `Note: False positive [...]` is parsed as `TestResults.Details.Websites.LikelyBlocked.Content.Paragraph`.

On a related note, is it correct to say that the JSON derives from the CSV? What is the process?

Reply:
As long as new lines are enclosed in `""` it will be parsed as a single string until the next `,` is found.

Yes, the `lang/en.json` file is derived from the CSV file. This is done by `scripts/update-translations.js`. The translation workflow is currently in a mess for `probe-desktop`. It currently uses an older version of a source CSV that was used by `probe-mobile` which has now moved onto what is available in `ooni/translations`. So there is a bunch of work to be done here to sync up the keys. Will open an issue for this.